invenio.cfg

```diff
@@ -114,15 +116,14 @@ THEME_SHOW_FRONTPAGE_INTRO_SECTION = False
 
 THEME_SITENAME = 'CDS'
 # Templates
-#THEME_FRONTPAGE_TEMPLATE = 'cds_rdm/frontpage.html'
-#THEME_FOOTER_TEMPLATE = 'cds_rdm/footer.html'
-#THEME_HEADER_TEMPLATE = 'cds_rdm/header.html'
+# THEME_FRONTPAGE_TEMPLATE = 'cds_rdm/frontpage.html'
```

Comment:
can be removed?

Reply:
Not sure, it was formatting which highlighted it as chane